std/net/punycode.d

```diff
@@ -0,0 +1,232 @@
+/**
+	Encodes and decodes punycode strings, per
+	$(LINK2 https://www.ietf.org/rfc/rfc3492.txt, RFC3492).
+	Punycode is primarily used for converting URIs or IRIs containing non-ASCII
+	characters into URIs or IRIs that are entirely ASCII, and vice versa.
+	This punycode codec is based upon the original implementation found in
+	$(LINK2 https://www.ietf.org/rfc/rfc3492.txt, RFC3492).
+
+	Authors: Shotaro Yamada (Sinkuu)
+	Date: February 27, 2017
+	License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+	Standards: $(LINK2 https://www.ietf.org/rfc/rfc3492.txt, RFC3492)
+	Source: $(PHOBOSSRC std/net/_punycode.d)
+	Version: 1.0.1
```

Comment:
`Date` and `Version` aren't commonly used as
- `date` get's automatically inserted based on the build
-  `version` = Phobos release, there are no subreleases

(also have a look at other modules for examples and the typical order)